playground/dist/index.html

```diff
@@ -105,21 +181,37 @@
       <div class="error-message" v-if="backendErrors.length" v-for="error in backendErrors">
         <i class="fas fa-exclamation-triangle"></i> {{ error.message }}
       </div>
-      <button class="open-code__button" v-show="!isCodeOpened" @click="openCode">
-        Open code to debug
-      </button>
-      <button class="open-code__button" v-show="isCodeOpened" @click="hideCode">
-        Hide code
-      </button>
-      <div v-if="isCodeOpened" class="debug-panel">
+      <div class="toolbar">
+        <div class="toolbar__title">
+          Weaverbird
+        </div>
+        <div class="toolbar__radio" :class="{ toolbar__selected: isSelectedTranslator('mongo50') }">
+          <a href="/">
+            MongoDB
+          </a>
+        </div>
+        <div class="toolbar__radio" :class="{ toolbar__selected:isSelectedTranslator('pandas') }">
+          <a href="?backend=pandas">
+            Pandas
+          </a>
+        </div>
+        <div class="toolbar__button" v-show="!isCodeOpened" @click="openCode">
+          Show code
+        </div>
+        <div class="toolbar__button" v-show="isCodeOpened" @click="hideCode">
+          Hide code
+        </div>
+      </div>
+      <div :class="{ opened: isCodeOpened }" class="debug-panel">
         <div class="debug-panel__code">
           <div class="debug-panel__code-pipeline">
             <div>Pipeline</div>
             <pre>{{ pipelineAsJSON }}</pre>
           </div>
           <div class="debug-panel__code-translated-pipeline">
             <div>Translated pipeline</div>
-            <pre>{{ mongoQueryAsJSON }}</pre>
+            <pre v-if="isSelectedTranslator('mongo50')">{{ mongoQueryAsJSON }}</pre>
+            <pre v-else>Query executed but translation not configured for this backend</pre>
```

Comment:
Indeed, translation makes no sense for pandas "executor".
I suggest you to hide completely translation panel: `<div class="debug-panel__code-translated-pipeline" v-if="!isSelectedTranslator('pandas')">`